packages/markdown/parseMarkdown.js

```diff
@@ -351,7 +363,9 @@ function createRender(context) {
 
       let finalHref = href;
 
-      checkUrlHealth(href, linkText, context);
+      if (process.env.DEPLOY_ENV !== 'production') {
+        checkUrlHealth(href, linkText, context);
+      }
```

Comment:
Actually, no, the content of this file is never sent to the users.

```suggestion
      checkUrlHealth(href, linkText, context);
```